lib/html_to_plain_text.rb

```diff
@@ -147,5 +149,9 @@ def format_list_item(out, options)
         out << "#{number}. "
       end
     end
+
+    def data_table?(table)
```

Comment:
This method seems mis-named since I would assume "data" tables would be the ones with borders.